src/middlewared/middlewared/plugins/vm/vms.py

```diff
@@ -40,6 +41,7 @@ class VMModel(sa.Model):
     ensure_display_device = sa.Column(sa.Boolean(), default=True)
     arch_type = sa.Column(sa.String(255), default=None, nullable=True)
     machine_type = sa.Column(sa.String(255), default=None, nullable=True)
+    uuid = sa.Column(sa.String(255), default=None, nullable=True)
```

Comment:
Doesn't `__common_validation` always set `uuid` so that the database column should never be `null`?